MT3620_M4_Sample_Code/FreeRTOS/MT3620_RTApp_FreeRTOS_CPP/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ add_link_options(-specs=nano.specs -specs=nosys.specs)
 
 # Executable
 add_executable(${PROJECT_NAME}
+               startup.cpp
```

Comment:
Please don't create the new file.
Please move the implementation of startup.cpp to main.cpp

Reply:
@LawranceLiu Modified.